lib/reversetunnel/srv_test.go

```diff
@@ -38,7 +38,7 @@ func TestServerKeyAuth(t *testing.T) {
 	require.NoError(t, err)
 
 	s := &server{
-		Entry: testlog.FailureOnly(t),
+		log: utils.NewLoggerForTests(),
```

Comment:
let's revert this change, `testlog.FailureOnly` seems appropriate for non-gocheck tests

Reply:
Yes, sorry - missed the spot.